app/scripts/directives/layout.dir.js

```diff
@@ -6,7 +6,13 @@ angular.module('yourStlCourts').directive('layout', function () {
 
     link: function ($scope, element) {
       $scope.$on('$stateChangeStart', function(event, toState){
-        if (toState.name === 'home' || toState.name === 'ticketSearch' || toState.name === 'about' || toState.name === 'help' || toState.name === 'info') {
+        var statesToUseBackground = ['home',
+                                     'ticketSearch',
+                                     'about',
+                                     'help',
+                                     'info',
+                                     'error'];
+        if (statesToUseBackground.indexOf(toState.name) != -1){
```

Comment:
Nice change! but can you put them on the same line? There isn't much value here in adding 5 lines of code where 1 will read just fine.